integration-tests/templates/rn76-template/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "ProjectName",
```

not exactly sure if this matters but to make it consistent with the other ones:
```suggestion
  "name": "rn76-template",
```

---

it doesn't matter because when you use the template and put a name there that one is used instead of what add this package.json file. I'll push the change just for consistency but it doesn't take any effect